src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -245,10 +245,9 @@ export class SamCliLocalInvokeInvocation {
             this.args.templatePath,
             '--event',
             this.args.eventPath,
-            '--env-vars',
-            this.args.environmentVariablePath,
```

Comment:
should we also do it for `--event` ?

Reply:
Yeah might as well at this point. Seems like I'll need to refactor the tests just to make them manageable (and facilitate later refactors to the impl.)